cli/tests/pcluster/config/test_source_consistency.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+
+import tests.pcluster.config.utils as utils
+from assertpy import assert_that
+from pcluster.config.mapping import ALIASES, AWS, CLUSTER, EBS, EFS, FSX, GLOBAL, RAID, SCALING, VPC
+from pcluster.config.pcluster_config import PclusterConfig
+from tests.pcluster.config.defaults import CFN_CLI_RESERVED_PARAMS, CFN_CONFIG_NUM_OF_PARAMS, DefaultCfnParams
+
+EXISTING_SECTIONS = [ALIASES, AWS, CLUSTER, EBS, EFS, FSX, GLOBAL, RAID, SCALING, VPC]
```

Comment:
I don't like it either, but I'd avoid a nested dict, because I'd have to perform a `CONFIG_SECTIONS.get()` every time I need to access the children, in code and tests.

I'd avoid this also because we don't need to retrieve all of them anywhere except this test.
This test is to verify source consistency not to verify code functionalities.